locales/de.json

```diff
@@ -1,2135 +1,2256 @@
 {
```

Comment:
the de.json shows a HUGE diff. that shouldn't be like this. what happened?

Reply:
lot of things changed because of merging conflict maybe didn't solve perfectly.

Reply:
The problem is you are removing keys, which were already existing.
I'd suggest du reset the de.json and just copy over your changes to resolve it good.